src/types.ts

```diff
@@ -15,6 +15,7 @@ export interface BibleBook {
   aliases: readonly string[];
   longName: string;
   shortName: string;
+  chapters?: number;
```

Comment:
since chapters are always defined this should not be optional imo.
The locale files could use `Omit<'chapters', BibleBook>`